docs/components/buttons.md

```diff
@@ -1,38 +1,190 @@
+
+## Types of buttons
+
+### Primary button
+
+This is the most common type of button. Use these to direct people to a clear primary action. To call out that these buttons are actionable, they are action blue (`{{ tokens.colors.action }}`).
+
+```html
+<button class="btn">
+  Do something
+</button>
+```
+
+Try not to have more than 1 primary button on a page. Multiple primary buttons puts more cognitive load on people. It takes them more time and effort to figure out what to do.
+
+### Inverse button
+
+On some backgrounds, the primary button color will not have sufficient contrast. In this case, use inverse buttons, which have a white fill.
+
+```html wrapper_class="bg-blue-dark p-20"
+<div class="bg-blue-dark">
+  <button class="btn btn-inverse">
+    Do something
+  </button>
+</div>
+```
+
+### Secondary button
+
+```html
+<button class="btn btn-secondary">
+  Do something
+</button>
+```
+
+Secondary buttons don’t have as much visual weight because they are outlined instead of solid. Use these if your button is not an important action on the page. 
+
+Pair it side-by-side with a primary button to prompt toward the primary button’s action.
+
+```html id="buttons-side-by-side"
+<div class="flex gap-20">
+  <button class="btn btn-secondary">
+    Cancel
+  </button>
+  <button class="btn">
+    Continue
+  </button>
+</div>
+```
+
+## Variations
+
+### Block buttons
+
+Block buttons stretch to fill the width of the screen or area instead of having a set width. They are often used on mobile layouts.
+
+```html
+<div class="bg-blue-1 p-20 text-slate text-body">
+  <div>COVID-19 vaccination appointments available only for San Francisco Health Network patients.</div>
+  <button class="btn btn-block my-20">
+    Check for availability
+  </button>
+  <div class="text-center">
+    Or call <a href="tel:1234567890" class="text-action">123-456-7890</a>
+  </div>
+</div>
+```
+
+### Icons
+
+Icons can be on the left or right of the text in a button. Do not use more than one icon in a button.
+
+```html
+<div class="flex justify-around items-center gap-16 title-xs">
+  <a href="#" class="btn btn-secondary flex gap-8">
+    <sfgov-icon symbol="arrow-left"></sfgov-icon>
+    <span>Previous</span>
+  </a>
+  <a href="#" class="text-action">1</a>
+  <span>...</span>
+  <a href="#" class="text-action">8</a>
+  <a href="#" class="text-action">9</a>
+  <span>10</span>
+  <a href="#" class="text-action">11</a>
+  <a href="#" class="btn btn-secondary flex gap-8">
+    <span>Next</span>
+    <sfgov-icon symbol="arrow-right"></sfgov-icon>
+  </a>
+</div>
+```
+
+Icons should reinforce the meaning of the button’s text. In rare cases, an icon can be used without text. Only do this if the icon is extremely universally understood, such as an arrow or search magnifying glass. [Read more about icons](/foundations/icons/)
+
+## Appearance
+
+<img src="/static/images/button-specs.png" class="w-full">
```

Comment:
I think this qualifies as a [decorative image](https://www.w3.org/WAI/tutorials/images/decorative/) since the text below explains the essence of the image, though we'll want an empty alt attribute (`alt=""`) so screen readers know to ignore the element. According to the W3C WAI guidelines, some screen readers will read the filename when an `alt` attribute is missing, which would be 🤢